scripts/build.sh

```diff
@@ -140,6 +141,15 @@ function run_check_style() {
     cargo fmt -- --check
 }
 
+function run_lit_test() {
+    # We need a binary to execute the lit tests
```

Comment:
There are debug and release flags we could use here, so that we could choose if we want to run one or both options (`./build.sh --lit` and `./build.sh --lit --release)

Reply:
Done, thanks for pointing this out